satellite/satellitedb/migrate.go

```diff
@@ -815,6 +815,13 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					);`,
 				},
 			},
+			{
+				Description: "Change type of disqualified column of nodes table to timestamp",
+				Version:     32,
+				Action: migrate.SQL{
+					`ALTER TABLE nodes ALTER COLUMN disqualified TYPE timestamp with time zone`,
```

Comment:
I assume that currently the `disqualified` field was not used, otherwise it should be done in different way (using several queries and alter statements) to set the current `disqualified = true` to something like `disqualified = <<some timestamp in the past>>` rather than setting all of them to `NULL`

Reply:
Never mind, I updated the migration query and it currently sets a specific timestamp for the nodes which could have the `disqualified` column set to `FALSE`